tests/src/registry-smoke.spec.ts

```diff
@@ -0,0 +1,116 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Page } from 'playwright';
+import { PodmanDesktopRunner } from './runner/podman-desktop-runner';
+import { afterAll, beforeAll, test, describe, beforeEach } from 'vitest';
+import { expect as playExpect } from '@playwright/test';
+import type { RunnerTestContext } from './testContext/runner-test-context';
+import { WelcomePage } from './model/pages/welcome-page';
+import { SettingsBar } from './model/pages/settings-bar';
+import { RegistriesPage } from './model/pages/registries-page';
+
+let pdRunner: PodmanDesktopRunner;
+let page: Page;
+let registryUrl: string;
+let registryUsername: string;
+let registryPswdSecret: string;
+let imageName: string;
+let imageTag: string;
+
+beforeAll(async () => {
+  pdRunner = new PodmanDesktopRunner();
+  page = await pdRunner.start();
+  pdRunner.setVideoName('registry-e2e');
+
+  registryUrl = '';
+  registryUsername = '';
+  registryPswdSecret = process.env.registryPswdSecret ? process.env.registryPswdSecret : 'invalidPswd';
+  imageName = '';
+  imageTag = '';
+
+  const welcomePage = new WelcomePage(page);
+  await welcomePage.handleWelcomePage(true);
+});
+
+afterAll(async () => {
+  await pdRunner.close();
+});
+
+beforeEach<RunnerTestContext>(async ctx => {
+  ctx.pdRunner = pdRunner;
+});
+
+describe('Registry workflow verification', async () => {
```

Comment:
Consider divide the suite into two, or two describes, one to test registry handling and the other adding authenticated registry and pulling a private image from it.